backend/routes/generations.py

```
@@ -3,51 +3,22 @@
-from pathlib import Path
 
-from fastapi import APIRouter, Depends, File, HTTPException, UploadFile
+from fastapi import APIRouter, Depends, HTTPException
 from fastapi.responses import StreamingResponse
 from sqlalchemy.orm import Session
 
-from .. import config, models
-from ..services import history, personality, profiles, tts
+logger = logging.getLogger(__name__)
+
+from .. import models
```

Ruff (0.15.12)

[error] Module level import not at top of file

(E402)